fars/booking/tests.py

```diff
@@ -1,3 +1,114 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from booking.models import Bookable, Booking
+from django.contrib.auth import get_user_model
+from datetime import datetime, timedelta
 
 # Create your tests here.
+class BookTestCase(TestCase):
+    def setUp(self):
+        Bookable.objects.create(
+            id_str = "b",
+            name = "B",
+            description = "A bookable"
+        )
+
+        User = get_user_model()
+        User.objects.create_user(
+            username = "u",
+            password = "pw"
+        )
```

Comment:
I'd recommend not creating test data in `setUp`, it will annoy you when you decide to test something that requires a different setup. It's better to call some simple functions at the start of each test.